Package.swift

```diff
@@ -36,7 +36,7 @@ let package = Package(
         .package(
             name: googleMobileAdsModuleName,
             url: "https://github.com/googleads/swift-package-manager-google-mobile-ads.git",
-            "9.0.0" ..< "11.0.0"
+            "11.0.0" ..< "13.0.0"
```

Comment:
Would not be better to have < 11.2.0?
How could we ensure compatibility with versions that are not yet released?

Reply:
Yes I do agree but we have to be careful to properly update this file every release then

Reply:
We at least have to make it so that they can still update the googleMobilAds file if it stays under the 12.0.0 version and under that one, normally there should be no breaking changes. If it is updates, it will be a security or minor bug update so we shouldn't have any compatibility issues until the next major update.